tests/boards/nrf/qdec/src/main.c

```diff
@@ -107,12 +107,16 @@ static void qenc_emulate_verify_reading(int emulator_period_ms, int emulation_du
 	k_msleep(emulation_duration_ms);
 
 	rc = sensor_sample_fetch(qdec_dev);
-	zassert_true(rc == 0, "Failed to fetch sample (%d)", rc);
+
+	if (!overflow_possible) {
+		zassert_true(rc == 0, "Failed to fetch sample (%d)", rc);
+	} else {
+		zassert_true(rc == -EOVERFLOW, "Failed to detect overflow");
```

Comment:
Is the overflow possible or expected? I think it is the latter and the variable name should be corrected, as well as the following comment: https://github.com/zephyrproject-rtos/zephyr/blob/d0d1cfc64644fc9b4f0722e1d87f6ff14eca55cb/tests/boards/nrf/qdec/src/main.c#L251-L253

Reply:
When running the test locally, it always led to overflow, so I agree